tensorflow/g3doc/api_docs/python/train.md

```diff
@@ -1332,6 +1332,7 @@ Wait till the Coordinator is told to stop.
 ### `class tf.train.QueueRunner` {#QueueRunner}
 
 Holds a list of enqueue operations for a queue, each to be run in a thread.
+It asynchronously runs the enqueue ops in different threads when the queuue is not full.
```

Comment:
1) queuue -> queue

2) I think to be precise, the thread always runs the enqueue op, but the enqueue op itself blocks when the queue is full.  @josh11b or @mrry to confirm